java-manta-examples/src/main/java/ClientEncryptionServerMultipart.java

```diff
@@ -52,7 +52,7 @@ public static void main(String... args) {
     }
 
     private static void multipartUpload(EncryptedServerSideMultipartManager multipart) {
-        String uploadObject = "/" + mantaUsername + "/stor/multipart";
+        String uploadObject = MantaClient.SEPARATOR + mantaUsername + "/stor/multipart";
```

Comment:
We should probably use the separator for the slashes in the `/stor/multipart` string as well.

Reply:
Done.